products/joomla.md

```diff
@@ -17,21 +17,21 @@ auto:
 # eol see https://developer.joomla.org/roadmap.html
 releases:
 -   releaseCycle: "5"
-    releaseDate: 2023-10-14
+    releaseDate: 2023-10-17
```

Comment:
You're correct about those `releaseDate` / `latestReleaseDate` dates, but because our automation is based on the git tag date we cannot change it: it will be reverted next time the automation is ran.